src/openforms/forms/management/commands/delete_export_files.py

```diff
@@ -0,0 +1,23 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management import BaseCommand
+from django.utils import timezone
+
+from ...models.form import FormsExport
+
+
+class Command(BaseCommand):
+    help = "Clear the downloaded form export files to free up disk space"
+
+    def handle(self, *args, **options):
+        before_date = (
+            timezone.now() - timedelta(days=settings.FORMS_EXPORT_REMOVED_AFTER_DAYS)
+        ).date()
+
+        forms_exports = FormsExport.objects.filter(
+            downloaded=True, date_downloaded__lt=before_date
+        )
+
+        for forms_export in forms_exports:
+            forms_export.delete()
```

Comment:
it's just a queryset, why not `forms_exports.delete()`?

Reply:
Because otherwise the `.delete()` on the model doesn't get called (which removes the files from the filesystem).

I could use the signals, but I thought that this would be simpler? :sweat_smile:

Reply:
Hrm, the code in `openforms.utils.files.DeleteFilesQuerySetMixin` should take care of that though :thinking:

Reply:
Ah I shall check again, maybe I made a mistake

Reply:
Yes, I made a mistake